ha/core/event_analyzer/filter/filter.py

```diff
@@ -181,19 +182,18 @@ def filter_event(self, msg: str) -> bool:
             msg (str): Msg
         """
         try:
-            K8S_alert_required = False
-            message = json.loads(msg)
-
-            event_namespace = message.get("namespace")
-            event_pod_name = message.get("pod_name")
-
-            namespace = Conf.get(const.HA_GLOBAL_INDEX, f"K8S:POD{_DELIM}namespace")
-            pod_name = Conf.get(const.HA_GLOBAL_INDEX, f"K8S:POD{_DELIM}pods")
-            if event_namespace == namespace and event_pod_name.startswith(pod_name):
-                K8S_alert_required = True
-                return K8S_alert_required
-            else:
-                return K8S_alert_required
-
+            resource_alert_required = False
+            message = json.dumps(ast.literal_eval(msg))
+            message = json.loads(message)
+
+            Log.debug('Received alert from fault tolerance')
+            event_resource_type = message.get("_resource_type")
+
+            required_resource_type = Conf.get(const.HA_GLOBAL_INDEX, f"NODE{_DELIM}resource_type")
```

Comment:
check if this can get in __init__ so no need to fetch again and again as this function will be called very frequently.

Reply:
This is a merge request to merge the existing code from fault_k8s branch to kubernetes branch
No code change should be done except for fixing conflicts/merge issues

Reply:
ok